schemas/aws/account-1.yml

```diff
@@ -112,6 +112,8 @@ properties:
     description: "Rosa related attributes in the aws account."
     "$ref": "/common-1.json#/definitions/crossref"
     "$schemaRef": "/dependencies/rosa-ocm-1.yml"
+  sso:
```

Comment:
Can you add a `description` field to explain what this does exactly?

Reply:
Added in [6b2a434](https://github.com/app-sre/qontract-schemas/pull/584/commits/6b2a4340fc38d7177a50e2d7f694c5908c7fbfd6)